addons/pos_self_order/static/src/overrides/components/receipt_header/receipt_header.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="pos_self_order.ReceiptHeader" t-inherit="point_of_sale.ReceiptHeader" t-inherit-mode="extension">
+        <xpath expr="//h1[hasclass('tracking-number')]" position="after">
+            <div t-if="props.data.pickingService" class="picking-service text-center">
+                <span t-if="props.data.pickingService == 'table'" >Service at Table</span>
+                <span t-else="">Pickup At Counter</span>
+                <br/>
```

Comment:
i know that we use `<br />` all the time in the receipt and it's not the worst thing, but normally we'd just using margin or padding